neqo-http3/tests/webtransport/mod.rs

```diff
@@ -456,23 +455,29 @@ impl WtTest {
     }
 
     fn receive_reset_server(&mut self, expected_stream_id: StreamId, expected_error: u64) {
-        assert!(matches!(
-            self.server.next_event().unwrap(),
-            Http3ServerEvent::StreamReset {
-                stream,
-                error
-            } if stream.stream_id() == expected_stream_id && error == expected_error
-        ));
+        let stream_reset = |e| {
+            matches!(
+                e,
+                Http3ServerEvent::StreamReset {
+                    stream,
+                    error
+                } if stream.stream_id() == expected_stream_id && error == expected_error
+            )
+        };
+        assert!(self.server.events().any(stream_reset));
```

Comment:
Can we find the exact event instead of using `any`?

Reply:
This will fine the exact event with expected steam_id(expected_stream_id) and expected_error.